config-model/pom.xml

```diff
@@ -53,7 +53,7 @@
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
-      <version>13.0.1</version>
+      <version>18.0</version>
```

Comment:
@gjoranv is this ok?

Reply:
No, we cannot upgrade guava until Vespa 7.

Reply:
We can get around that, as we can have multiple versions loaded at the same time in separate OSGi bundles. Perhaps better done in a separate PR though.